Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.h

```diff
@@ -240,6 +241,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong, nonnull) BUYModelManager *modelManager;
 
 /**
+ *  Queue where on which all request operation are executed
+ */
+@property (nonatomic, strong, readonly, nonnull) NSOperationQueue *requestQueue;
```

Comment:
for consistency, lets type the callback queue to an NSOperation, and name it `callbackQueue` for clarity.

then when you need to callback, you can:
```
		[callbackQueue addOperationWithBlock: ^{
			completionHandler(json, response, error);
		}];
```